### PR TITLE
Ichor redone.

### PR DESCRIPTION
**Pull request by Abatha**
## About The Pull Request
Redid the ichor coding to make it cleaner at Cimika's request.Probably fucked something up again with this, if I did I'll need to be told what it was to fix it.
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -370,6 +383,19 @@
 	splittable = FALSE	// Is the stack capable of being splitted?
 	preloaded_reagents = list("silicon" = 4, "ethanol" = 10, "mercury" = 4)
 
+/obj/item/stack/medical/advanced/ointment/regenerative_ichor
+	name = "regenerative ichor"
+	singular_name = "regenerative ichor"
+	desc = "An ichor that regenerates dead cells and fights off bacterial infection."
+	icon_state = "regenerative_ichor"
+	heal_burn = 8
+	origin_tech = list(TECH_BIO = 2)
```
Comment (KathyRyals) [inline code/game/objects/items/stacks/medical.dm:392]: Already defined in parent type, no need to add it here.

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -370,6 +383,19 @@
...
+	automatic_charge_overlays = FALSE
+	consumable = TRUE	// Will the stack disappear entirely once the amount is used up?
+	splittable = FALSE	// Is the stack capable of being splitted?
+	preloaded_reagents = list("silicon" = 4, "ethanol" = 10, "mercury" = 4)
```
Comment (KathyRyals) [inline code/game/objects/items/stacks/medical.dm:396]: Same.

File: code/game/objects/items/stacks/medical.dm [hunk 3]
```diff
@@ -274,6 +274,19 @@
 	splittable = FALSE	// Is the stack capable of being splitted?
 	preloaded_reagents = list("silicon" = 4, "ethanol" = 10, "lithium" = 4)
 
+/obj/item/stack/medical/advanced/bruise_pack/mending_ichor
+	name = "mending ichor"
+	singular_name = "mending ichor"
+	desc = "An ichor that can be used to mend physical trauma."
+	icon_state = "mending_ichor"
+	heal_brute = 8
+	origin_tech = list(TECH_BIO = 2)
```
Comment (KathyRyals) [inline code/game/objects/items/stacks/medical.dm:283]: Already defined in parent type. That means your mending_ichor type will already inherits these values from the bruise_pack parent, so you don't have to define them.

File: code/game/objects/items/stacks/medical.dm [hunk 4]
```diff
@@ -274,6 +274,19 @@
...
+	automatic_charge_overlays = FALSE
+	consumable = TRUE	// Will the stack disappear entirely once the amount is used up?
+	splittable = FALSE	// Is the stack capable of being splitted?
+	preloaded_reagents = list("silicon" = 4, "ethanol" = 10, "lithium" = 4)
```
Comment (KathyRyals) [inline code/game/objects/items/stacks/medical.dm:287]: Same.

**Review by KathyRyals — CHANGES_REQUESTED** [review]

**Review by KathyRyals — APPROVED** [review]

**Comment by Kazkin** [comment]
This is much cleaner and significantly improved compared to the first draft, while frustrating, this code is as clean as my own. Approved.

**Pull request merged**